src/go/rpk/pkg/cli/cmd/wasm/deploy.go

```diff
@@ -0,0 +1,106 @@
+package transform
+
+import (
+	"encoding/binary"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"vectorized/pkg/utils"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewDeployCommand(
+	fs afero.Fs, createProduce func(bool, int32) (sarama.SyncProducer, error),
+) *cobra.Command {
+	var (
+		description	string
+		jvmPartitioner	bool
+		partition	int32
+	)
+
+	command := &cobra.Command{
+		Use:	"deploy <path>",
+		Short:	"deploy inline WASM function",
+		Args:	cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			path := args[0]
+			fullFileName := filepath.Base(path)
+			fileExt := filepath.Ext(fullFileName)
+			// validate file extension, just allow js extension
+			if fileExt != ".js" {
+				return fmt.Errorf("deploy just allow js extension file")
+			}
+			fileName := strings.Replace(fullFileName, fileExt, "", 1)
+			fileContent, err := utils.ReadFileLinesBinary(fs, path)
+			if err != nil {
+				return err
+			}
+			// create producer
+			producer, err := createProduce(jvmPartitioner, partition)
```

Comment:
Please double check, but I think the internal topic for coproc will have only one partition.